
From: Christoph Hellwig <hch@lst.de>

<linux/irq.h> is internals for the generic irq handler implementation,
which is used on most but not all platforms.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/ieee1394/ohci1394.c         |    1 -
 25-akpm/drivers/ieee1394/pcilynx.c          |    1 -
 25-akpm/sound/isa/sgalaxy.c                 |    2 +-
 25-akpm/sound/oss/dmasound/dmasound_awacs.c |    1 -
 4 files changed, 1 insertion(+), 4 deletions(-)

diff -puN drivers/ieee1394/ohci1394.c~dont-include-linux-irqh-from-drivers drivers/ieee1394/ohci1394.c
--- 25/drivers/ieee1394/ohci1394.c~dont-include-linux-irqh-from-drivers	2004-10-01 20:16:47.955778832 -0700
+++ 25-akpm/drivers/ieee1394/ohci1394.c	2004-10-01 20:16:47.966777160 -0700
@@ -94,7 +94,6 @@
 #include <linux/pci.h>
 #include <linux/fs.h>
 #include <linux/poll.h>
-#include <linux/irq.h>
 #include <asm/byteorder.h>
 #include <asm/atomic.h>
 #include <asm/uaccess.h>
diff -puN drivers/ieee1394/pcilynx.c~dont-include-linux-irqh-from-drivers drivers/ieee1394/pcilynx.c
--- 25/drivers/ieee1394/pcilynx.c~dont-include-linux-irqh-from-drivers	2004-10-01 20:16:47.957778528 -0700
+++ 25-akpm/drivers/ieee1394/pcilynx.c	2004-10-01 20:16:47.967777008 -0700
@@ -42,7 +42,6 @@
 #include <linux/pci.h>
 #include <linux/fs.h>
 #include <linux/poll.h>
-#include <linux/irq.h>
 #include <linux/kdev_t.h>
 #include <asm/byteorder.h>
 #include <asm/atomic.h>
diff -puN sound/isa/sgalaxy.c~dont-include-linux-irqh-from-drivers sound/isa/sgalaxy.c
--- 25/sound/isa/sgalaxy.c~dont-include-linux-irqh-from-drivers	2004-10-01 20:16:47.958778376 -0700
+++ 25-akpm/sound/isa/sgalaxy.c	2004-10-01 20:16:47.968776856 -0700
@@ -26,7 +26,7 @@
 #include <linux/init.h>
 #include <linux/delay.h>
 #include <linux/time.h>
-#include <linux/irq.h>
+#include <linux/interrupt.h>
 #include <linux/moduleparam.h>
 #include <sound/core.h>
 #include <sound/sb.h>
diff -puN sound/oss/dmasound/dmasound_awacs.c~dont-include-linux-irqh-from-drivers sound/oss/dmasound/dmasound_awacs.c
--- 25/sound/oss/dmasound/dmasound_awacs.c~dont-include-linux-irqh-from-drivers	2004-10-01 20:16:47.960778072 -0700
+++ 25-akpm/sound/oss/dmasound/dmasound_awacs.c	2004-10-01 20:16:47.970776552 -0700
@@ -76,7 +76,6 @@
 #include <linux/nvram.h>
 #include <linux/tty.h>
 #include <linux/vt_kern.h>
-#include <linux/irq.h>
 #include <linux/spinlock.h>
 #include <linux/kmod.h>
 #include <linux/interrupt.h>
_
